packaging/alvr_build_linux.sh

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env bash
```

Comment:
Can you run shellcheck on this? There's a lot of [things it shows](https://i.ckie.dev/WwztUj0.txt).